reflection/app/Http/Controllers/EmployeeController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Employee;
+use App\Models\Company;
+
+class EmployeeController extends Controller
+{
+    public function index($id)
```

Comment:
You could use route model bindings  here, then use the relationship to get employees.
e.g.
```
$employees = $company->employees()->paginate(10);
```